test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -259,10 +280,7 @@ namespace iroha {
                        .build();
       auto result = executeQuery(query);
       ASSERT_NO_THROW({
-        const auto &cast_resp = boost::apply_visitor(
-            framework::SpecifiedVisitor<
-                shared_model::interface::AccountResponse>(),
-            result->get());
+        const auto &cast_resp = getRef<AccountResponse>(result->get());
```

Comment:
Please, cancel changes in existing tests, as there will be a significant amount of conflicts with this [PR](https://github.com/hyperledger/iroha/pull/1901) when merging